mupdf-sys/src/lib.rs

```diff
@@ -4,3 +4,13 @@
 #![allow(clippy::all)]
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+pub unsafe fn mupdf_new_derived_device<T>(
```

Comment:
I think this fn should have some more documentation about safety or bounds for `T`